src/components/ConfirmationModalDialog/ConfirmationModalDialog.vue

```diff
@@ -25,15 +25,17 @@
       <div class="actions" :data-testid="`${dataTestId}-actions`">
         <button v-if="dismissButtonText"
                 class="action dismiss"
-                :disabled="disabled"
+                :class="{ disabled: dismissButtonDisabled }"
```

Comment:
(nit) this extra `disabled` class is unnecessary since we could just use the `:disabled` CSS pseudo-selector instead. Again, no need to change it right now, but please keep it in mind for the future.

Same comment for line: 37, below.